README.md

```diff
@@ -5,6 +5,8 @@ This is not an officially supported Google product.
 This tool validates selected [GKE](https://cloud.google.com/kubernetes-engine) cluster
 against set of [REGO policies](https://www.openpolicyagent.org/docs/latest/policy-language/).
 
+Please check out [Contributing](https://github.com/google/gke-policy-automation/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/google/gke-policy-automation/blob/main/docs/code-of-conduct.md) docs before contributing.
```

Comment:
Adding relative links instead of URLs makes more sense here for me
i.e.
```
[Contributing](./CONTRIBUTING.md)
```

Reply:
yeah, totally agree, it was just to test the cla/google integration on the PR ;) let me fix this